k8s/prod/common/reform-scan/blob-router.yaml

```diff
@@ -21,6 +21,7 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/reform-scan/blob-router:prod-d9a7fbc1
       environment:
+        CRIME_ENABLED: true
```

Comment:
Alternatively, we could just have crime enabled by default. Thoughts?

Reply:
Crime processing won’t work on prod at the moment so better not to enable it for now ?

We could enable it by default in values yaml and disable for prod in flux ?

Reply:
if not going to work on prod why are we enabling then here?

Reply:
Why is it not going to work in prod?